source/integration-guide/advanced.rst

```diff
@@ -327,3 +327,68 @@ To enable the purchase history tooltip add the ``tooltipEnabled`` option to the
         tooltipEnabled: true,
         tooltipStyle: "light"
     });
+
+
+Add to cart
+^^^^^^^^^^^
+You can enable an add to cart button directly inside the Virtusize widget.
+
+While it is not required, we recommend that you provide Virtusize with a list
+of **available sizes** when you enable add to cart. If you do so, the add to
+cart button will be disabled and a little note will inform your customer for
```

Comment:
reword - If you do so, for items that are out of stock, the add to cart button will be disabled and a note will inform your customer that the item is not available.